stepBuild.go

```diff
@@ -124,7 +124,7 @@ func (c *StepBuild) GetCommand() *cobra.Command {
 				}
 
 				ColorPrintln("fix", Magenta)
-				if err := Exec("go", "fix", "./..."); err != nil {
+				if err := ExecShell("go tool fix $(go list -e -f '{{.Dir}}' ./...)"); err != nil {
```

Comment:
You still need specify the go version, no? IMO, to workaround the issue, there should be something like this:
```go
var err = null
var minGoVersion = ... // the version go.mod's go directive, e.g. "1.22.0"
if strings.HasSuffix(minGoVersion, ".0") {
	minGoVersion = strings.TrimSuffix(minGoVersion, ".0") // e.g. "1.22"
        err = ExecShell("go tool fix -go=go" + goVersion + " $(go list -e -f '{{.Dir}}' ./...)")
} else {
	err = Exec("go", "fix", "./...")
}
if err != nil { // ...
```